alsa-lib/README.md

```diff
@@ -16,3 +16,25 @@ Typically this is a runtime dependency that can be added to your
 plan.sh:
```

Comment:
We're missing usage for the `bin/aserver` bin that this plan provides - but I cannot easily find any documentations or examples for it's usage. The library files seem to be the payload of this plan.